app/experimenter/legacy-ui/rapid/components/experiments/ExperimentDetails.tsx

```diff
@@ -118,7 +119,18 @@ const ExperimentDetails: React.FC = () => {
         <h2 className="font-weight-bold">Results</h2>
         {experimentData.analysis?.show_analysis ? (
           <div>
-            <ResultsTable experimentData={experimentData} />
+            <ResultsTable {...{ experimentData }} />
+            <div>
+              {experimentData.features.map((feature) => {
+                return (
+                  <PrimaryMetricTable
+                    key={feature}
```

Comment:
Is this a whole object?  Maybe it should be something atomic like a string identifier or something?

Reply:
it's a string

Reply:
Oh okay cool 👍

Reply:
I wish I could see the TS types from GH somehow....